pkg/kubectl/cmd/set/set_resources.go

```diff
@@ -120,6 +120,7 @@ func NewCmdResources(f cmdutil.Factory, out io.Writer, errOut io.Writer) *cobra.
 	cmd.Flags().BoolVar(&options.Local, "local", false, "If true, set resources will NOT contact api-server but run locally.")
 	cmdutil.AddDryRunFlag(cmd)
 	cmdutil.AddRecordFlag(cmd)
+	cmdutil.AddIncludeUninitializedFlag(cmd)
```

Comment:
We need to update the comment for` -l` and `--all` for this command as well, right?

Reply:
Correct. Done. PTAL.